coord.Rmd

```diff
@@ -159,7 +159,7 @@ Internally ggplot2 uses many more segments so that the result looks smooth.
 
 ### Transformations with `coord_trans()`
 
-Like limits, we can also transform the data in two places: at the scale level or at the coordinate system level. `coord_trans()` has arguments `x` and `y` which should be strings naming the transformer or transformer objects (see [continous position scales](#sub:scale-position)). Transforming at the scale level occurs before statistics are computed and does not change the shape of the geom. Transforming at the coordinate system level occurs after the statistics have been computed, and does affect the shape of the geom. Using both together allows us to model the data on a transformed scale and then backtransform it for interpretation: a common pattern in analysis. \index{Transformation!coordinate system} \index{Coordinate systems!transformed} \indexf{coord\_trans}
+Like limits, we can also transform the data in two places: at the scale level or at the coordinate system level. `coord_trans()` has arguments `x` and `y` which should be strings naming the transformer or transformer objects (see [continous position scales](#scale-position)). Transforming at the scale level occurs before statistics are computed and does not change the shape of the geom. Transforming at the coordinate system level occurs after the statistics have been computed, and does affect the shape of the geom. Using both together allows us to model the data on a transformed scale and then backtransform it for interpretation: a common pattern in analysis. \index{Transformation!coordinate system} \index{Coordinate systems!transformed} \indexf{coord\_trans}
```

Comment:
Also need to update `[continous position scales](#scale-position)` to `Section \@ref(scale-position)`.  I _think_ it's easiest to do that in this PR, since you also need to touch those refs for the new labels.

Reply:
Ah, got it and yes I agree. I misread the issue! Will take care of that now